lib/livereload.js

```diff
@@ -97,6 +97,12 @@ function StartServer() {
                         return monkeyPatch() + snippet + match;
                     }
                 }
+            },
+            minify: false,
```

Comment:
What fixes the bug is the 'minify' being set to false. minification was the cause behind the breakage.
I guess the description might not have been clear enough.